server/repository/src/db_diesel/temperature_breach.rs

```diff
@@ -35,12 +34,6 @@ pub struct TemperatureBreachFilter {
     pub location: Option<LocationFilter>,
 }
 
-impl EqualFilter<TemperatureBreachRowType> {
-    pub fn equal_to_breach_type(value: &TemperatureBreachRowType) -> Self {
```

Comment:
Changes in this file are extra, just did the minimum for the filters (and existing pattern)